packages/codegen-ui-react/lib/forms/form-renderer-helper/bidirectional-relationship.ts

```diff
@@ -156,80 +277,14 @@ export function getBiDirectionalRelationshipStatements({
-                          ),
-                          undefined,
-                          [
-                            factory.createIdentifier(relatedRecordToUnlink),
-                            factory.createArrowFunction(
-                              undefined,
-                              undefined,
-                              [
-                                factory.createParameterDeclaration(
-                                  undefined,
-                                  undefined,
-                                  undefined,
-                                  factory.createIdentifier('updated'),
-                                  undefined,
-                                  undefined,
-                                  undefined,
-                                ),
-                              ],
-                              undefined,
-                              factory.createToken(SyntaxKind.EqualsGreaterThanToken),
-                              factory.createBlock(
-                                [
-                                  factory.createExpressionStatement(
-                                    factory.createBinaryExpression(
-                                      factory.createPropertyAccessExpression(
-                                        factory.createIdentifier('updated'),
-                                        factory.createIdentifier(fieldBiDirectionalWithName),
-                                      ),
-                                      factory.createToken(SyntaxKind.EqualsToken),
-                                      factory.createIdentifier('undefined'),
-                                    ),
-                                  ),
-                                  ...associatedFieldsBiDirectionalWith.map((field) =>
-                                    factory.createExpressionStatement(
-                                      factory.createBinaryExpression(
-                                        factory.createPropertyAccessExpression(
-                                          factory.createIdentifier('updated'),
-                                          factory.createIdentifier(field),
-                                        ),
-                                        factory.createToken(SyntaxKind.EqualsToken),
-                                        factory.createIdentifier('undefined'),
-                                      ),
-                                    ),
-                                  ),
-                                ],
-                                true,
-                              ),
-                            ),
-                          ],
-                        ),
-                      ],
-                    ),
-                  ],
-                ),
-              ),
+              isRelatedModelRequired
+                ? unlinkModelThrowErrorExpression(relatedModelName, relatedRecordToUnlink, modelName)
```

Comment:
Will ping you to find out what this is doing

Reply:
Yup will do